src/type_builder.ml

```diff
@@ -0,0 +1,65 @@
+type t
+
+type struct_field = {
+  type_ : Type.t;
+  packed_type : Packed_type.t;
+  mutable_ : bool;
+}
+
+type error =
```

Comment:
Might be a good idea to link to the values here: https://github.com/WebAssembly/binaryen/blob/64ba23996a10e229d46e41eb37736a55af87f79a/src/binaryen-c.h#L3618 

Just for future updates.

Might also be a good idea to document the update practice for this in #252